src/components/SmallDirectoryBlock.tsx

```
@@ -0,0 +1,173 @@
+} from "@mantine/core";
+import {
+  IconSearch,
+  IconChevronDown,
+  IconChevronUp,
+  IconArrowsVertical,
+  IconAlertCircle
+} from "@tabler/icons-react";
+import { useAttendees } from "@/hooks/attendees/useAttendees";
+import Profile from "@/assets/icons/Profile.svg";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
```

ESLint

[error] 'useRouter' is defined but never used.

(@typescript-eslint/no-unused-vars)